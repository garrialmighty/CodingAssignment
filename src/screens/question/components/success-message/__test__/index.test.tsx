import React from 'react';
import {render} from '@testing-library/react-native';

import SuccessMessage from '../';

describe('Component: SuccessMessage', () => {
  it('should render properly', () => {
    const snapshot = render(<SuccessMessage points={99} />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
