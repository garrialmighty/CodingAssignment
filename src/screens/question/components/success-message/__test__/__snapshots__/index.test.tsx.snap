// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: SuccessMessage should render properly 1`] = `
<View
  style={
    {
      "alignContent": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      {
        "color": "#769a8e",
        "fontSize": 50,
        "textAlign": "center",
      }
    }
  >
    Correct!
  </Text>
  <Text
    style={
      {
        "color": "#769a8e",
        "fontSize": 50,
        "textAlign": "center",
      }
    }
  >
    Congratulations
  </Text>
  <Text
    style={
      {
        "color": "#769a8e",
        "fontSize": 50,
        "textAlign": "center",
      }
    }
  >
    
You earn 99
points
  </Text>
</View>
`;
